<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recipeExtrasWidget</class>
 <widget class="QWidget" name="recipeExtrasWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_brewer">
       <property name="text">
        <string>Brewer</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_brewer</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="BtStringEdit" name="lineEdit_brewer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">brewer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_asstBrewer">
       <property name="text">
        <string>Asst. Brewer</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_asstBrewer</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_tasteRating">
       <property name="text">
        <string>Taste Rating</string>
       </property>
       <property name="buddy">
        <cstring>spinBox_tasteRating</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_tasteRating">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="BtStringEdit" name="lineEdit_asstBrewer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">asstBrewer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="BtTimeLabel" name="label_primaryAge">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Primary Age (days)</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_primaryAge</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="BtTimeEdit" name="lineEdit_primaryAge">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">primaryAge_days</string>
       </property>
       <property name="forcedScale" stdset="0">
        <string notr="true">scaleLarge</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="BtTemperatureLabel" name="label_primaryTemp">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Primary Temp</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_primaryTemp</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="BtTemperatureEdit" name="lineEdit_primaryTemp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">primaryTemp_c</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="BtTimeLabel" name="label_secAge">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Secondary Age (days)</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_secAge</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="BtTimeEdit" name="lineEdit_secAge">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">secondaryAge_days</string>
       </property>
       <property name="forcedScale" stdset="0">
        <string notr="true">scaleLarge</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="BtTemperatureLabel" name="label_secTemp">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Secondary Temp</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_secTemp</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="BtTemperatureEdit" name="lineEdit_secTemp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">secondaryTemp_c</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="BtTimeLabel" name="label_tertAge">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Tertiary Age (days)</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_tertAge</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="BtTimeEdit" name="lineEdit_tertAge">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">tertiaryAge_days</string>
       </property>
       <property name="forcedScale" stdset="0">
        <string notr="true">scaleLarge</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="BtTemperatureLabel" name="label_tertTemp">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Tertiary Temp</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_tertTemp</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="BtTemperatureEdit" name="lineEdit_tertTemp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">tertiaryTemp_c</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="BtTimeLabel" name="label_age">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Bottle/Keg Age (days)</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_age</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="BtTimeEdit" name="lineEdit_age">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">age</string>
       </property>
       <property name="forcedScale" stdset="0">
        <string notr="true">scaleLarge</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="BtTemperatureLabel" name="label_ageTemp">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="text">
        <string>Bottle/Keg Temp</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_ageTemp</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="BtTemperatureEdit" name="lineEdit_ageTemp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">ageTemp_c</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_date">
       <property name="text">
        <string>Date First Brewed</string>
       </property>
       <property name="buddy">
        <cstring>dateEdit_date</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDateEdit" name="dateEdit_date">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2011</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>dd MMM yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labe_carbVols">
       <property name="text">
        <string>Carbonation Volumes</string>
       </property>
       <property name="buddy">
        <cstring>lineEdit_carbVols</cstring>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="BtGenericEdit" name="lineEdit_carbVols">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="editField" stdset="0">
        <string notr="true">carbonation_vols</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Taste Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="BtTextEdit" name="btTextEdit_tasteNotes">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Notes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="BtTextEdit" name="btTextEdit_notes">
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtGenericEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtTimeEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTimeLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureEdit</class>
   <extends>QLineEdit</extends>
   <header>BtLineEdit.h</header>
   <slots>
    <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTemperatureLabel</class>
   <extends>QLabel</extends>
   <header>BtLabel.h</header>
   <slots>
    <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">BtTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>BtStringEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">BtLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_brewer</tabstop>
  <tabstop>lineEdit_asstBrewer</tabstop>
  <tabstop>spinBox_tasteRating</tabstop>
  <tabstop>lineEdit_primaryAge</tabstop>
  <tabstop>lineEdit_primaryTemp</tabstop>
  <tabstop>lineEdit_secAge</tabstop>
  <tabstop>lineEdit_secTemp</tabstop>
  <tabstop>lineEdit_tertAge</tabstop>
  <tabstop>lineEdit_tertTemp</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>lineEdit_ageTemp</tabstop>
  <tabstop>dateEdit_date</tabstop>
  <tabstop>lineEdit_carbVols</tabstop>
  <tabstop>btTextEdit_tasteNotes</tabstop>
  <tabstop>btTextEdit_notes</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>label_ageTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_ageTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_secTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_secTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_tertTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_tertTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_primaryTemp</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_primaryTemp</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_secAge</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_secAge</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_tertAge</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_tertAge</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_age</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_age</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>label_primaryAge</sender>
   <signal>labelChanged(Unit::unitDisplay,Unit::unitScale)</signal>
   <receiver>lineEdit_primaryAge</receiver>
   <slot>lineChanged(Unit::unitDisplay,Unit::unitScale)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
